### ISSUE TITLE
Support iTerm 2.9̅ Inline Images

### ISSUE THREAD

**Issue by eddieantonio** [eddieantonio/imgcat#2]
[Inline Images](https://iterm2.com/images.html). Also, their script is _also_ called `imgcat`... soooo....

**Comment by eddieantonio** [comment]
Incorporate this:

```
Add support for a custom CSI code: CSI 1337 n. We'll report a string of 'CSI
  ITERM2 versionnumber n'. This is useful for detecting that iTerm2 version 3 is
  running in a way that's compatible with other terminals.
```

**Comment by eddieantonio** [comment]
Fixed as of 2.0.0
